src/components/step/step.css

```diff
@@ -0,0 +1,54 @@
+html {
+    font-size: 17px;
+    background-color: #FFFFFF;
+}
+
+@supports (font: -apple-system-body) {
+    html {
+        font: -apple-system-body;
+    }
+}
+
+body,div,dl,dt,dd,ul,ol,li,h1,h2,h3,h4,h5,h6,pre,form,fieldset,input,textarea,p,blockquote,th,td { 
```

Comment:
We should ideally isolate this - the CSS is all bundled together, so this will override styles for all components (even not steps).  So I'd nest all the styles in this file under .step-layout if they are necessary.

The html/font stuff is already set up by https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/components/presentational/Layout/Layout.css...which probably should also be fixed to be more isolated now (actually not sure if it's possible to add a class to the <html> element, and I think it might be necessary to apply a style to ```<html>``` achieve dynamic type sizing...not sure)

For now though I'd probably just try to keep things isolated under the class as much as possible so we don't override styles for the rest of the view library downstream for instance

Reply:
I *think* that literally all the global styles should be defined in https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/components/presentational/Layout/Layout.css

Those that match we can remove presumably (which is good number of them) and I don't think I see any major conflicts between html defined there vs. here

Major issue might be the background color - might look at the Layout component to see how I solved that (there is a way to explicitly set it) and just explicitly set body background color in the StepLayout to #FFFFFF

But yeah, all the CSS is bundled together so have to be careful about global styles since I imagine this is overridding stuff in the view library now

Reply:
I've cleaned this up a bit but still a bit stymied on the html background color once you run the bundle in MDHViewLibrary. I can override the background var in the StepLayout.css like this:

```
:root
{
    --main-bg-color: #FFFFFF;
}
```

But that still ends up overriding the same var in the views so you end up with the survey white background there too. Only way I've seen that might work is if the StepLayout component actually did something like this to update the var at runtime:

`document.documentElement.style.setProperty('--main-bg-color', randomColor);`

Any other ideas?

Reply:
The second bit of that may not be working as intended:
`document.body.style.removeProperty('background-color');`

Seems like it may affect only inline styles.

Reply:
Gotcha, I think I probably never tested that since I'm not really removing the background color typically once the page is rendered...probably should be a default there anyways or something